velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -25,6 +25,11 @@ inline void registerArrayMinMaxFunctions() {
   registerFunction<ArrayMaxFunction, T, Array<T>>({"array_max"});
 }
 
+template <typename OT, typename IT>
+inline void registerArraySumFunction() {
+  registerFunction<ArraySumFunction, OT, IT>({"array_sum_alt"});
```

Comment:
@jwyles-ahana I assume this function need to be registered only in the tests or benchmarks. Do we need to be registered in production code? I assume not.

CC: @majetideepak

Reply:
Yes, it only needs to be registered for use in the benchmark.